pkg/session/session.go

```diff
@@ -301,9 +301,20 @@ func (s *Session) GetMessages(a *agent.Agent) []chat.Message {
 			validAgentIDs = append(validAgentIDs, agent.Name())
 		}
 
+		handoffPrompt := "You are part of a multi-agent team. Your goal is to answer the user query in the most helpful way possible.\n\n" +
+			"Available agents in your team:\n" + agentsInfo + "\n" +
+			"You can hand off the conversation to any of these agents at any time by using the `handoff` function with their ID. " +
+			"The valid agent names are: " + strings.Join(validAgentIDs, ", ") + ".\n\n" +
```

Comment:
We are talking about agent IDs above, let's keep it consistent
```suggestion
			"The valid agent IDs are: " + strings.Join(validAgentIDs, ", ") + ".\n\n" +
```